Lib/test/test_typing.py

```diff
@@ -6124,8 +6130,6 @@ def test_errors(self):
 
 
 # We need this to make sure that `@no_type_check` respects `__module__` attr:
```

Comment:
this comment seems orphaned now?

Reply:
I think it still makes sense referring to the classes immediately below